packages/phone-number-privacy/combiner/src/common/crypto-clients/domain-crypto-client.ts

```diff
@@ -0,0 +1,43 @@
+import { ErrorMessage, PoprfCombiner } from '@celo/phone-number-privacy-common'
+import Logger from 'bunyan'
+import { OdisConfig } from '../../config'
+import { CryptoClient } from './common'
+
+export class DomainCryptoClient extends CryptoClient {
+  private cryptoClient: PoprfCombiner
```

Comment:
No that's a good callout, missed this when I changed the name of the abstract class -> CryptoClient :) may just change to poprfCombiner since this class implementation is just a wrapper around that specific lib anyways